cmd/lk/utils.go

```diff
@@ -96,6 +98,34 @@ func withDefaultClientOpts(c *config.ProjectConfig) []twirp.ClientOption {
 	return opts
 }
 
+func extractArg(c *cli.Command) (*string, error) {
+	if !c.Args().Present() {
+		return nil, errors.New("no argument provided")
+	}
+	return &c.Args().Slice()[0], nil
```

Comment:
Isn't there an `c.Args().First()` helper?

Reply:
🤦

Reply:
OK, I believe my thinking was that you can't return `nil` in place of a `string`, so would have to `return "", err` in case of an error -- which felt wrong? IDK I am still learning what idiomatic Go is 😆

Reply:
Returning empty string is totally OK in go :+1: